Assets/MixedRealityToolkit-Preview/LightEstimation-Examples/Scripts/DebugView.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Preview.Examples.Demos
```

Comment:
this could become the basis of adding debug output as part of the Diagnostics System!